src/test/java/com/uid2/shared/attest/UidCoreClientTest.java

```diff
@@ -59,6 +55,37 @@ public void Download_Succeed_RequestSentWithExpectedParameters() throws IOExcept
         when(mockHttpClient.get("https://download", expectedHeaders)).thenReturn(mockHttpResponse);
 
         uidCoreClient.download("https://download");
+        verify(mockAttestationTokenRetriever, times(1)).attest();
+        verify(mockHttpClient, times(1)).get("https://download", expectedHeaders);
+    }
+
+    @Test
+    public void Download_With_Attest_Uses_JWT_Second_Call() throws IOException, CloudStorageException, AttestationTokenRetrieverException {
```

Comment:
Can the method name be `DownloadWithAttest_UsesJWTSecondCall()`? Or something along those lines

Reply:
Fixed